features/host_can_view_navbar.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Host can view navbar and see "Become a Host" link 
+    As a Host 
+    In order to use the service
+    I would like to navigate the site
+
+  @javascript
+  Scenario: Visitor can see the landing page
+    When I visit the landing page
+    Then I should see a navigation bar
+    And I should see "Welcome Admin"
```

Comment:
?? Is this a default display?

Reply:
we changed it to 'welcome' for when devise is ready. it should say 'welcome + user.email'